build/scripts/build-utils.ps1

```diff
@@ -353,6 +359,19 @@ function Get-MSBuildDir([switch]$xcopy = $false) {
     return $both[1]
 }
 
+
+# Dose this version of Visual Studio meet our minimum requirements for building.
+function Test-SupportedVisualStudioVersion([string]$version) { 
+    if (-not ($version -match "^([\d.]+)(\+|-)?.*$")) { 
+        Write-Host "here"
```

Comment:
I suspect this is leftover debugging detritus?

Reply:
Yep. Removed.